switch.py

```diff
@@ -0,0 +1,325 @@
+
+    # merge envs with same name but different accounts
+    # Group by name and combine accounts
+    grouped_envs = defaultdict(lambda: {"team": "", "created_by": "", "env_type": "", "state": "", "accounts": set()})
+
+    for env in envs:
+        key = env["name"].strip() or "N/A"
+        grouped_envs[key]["team"] = env.get("team", "").strip() or "N/A"
+        grouped_envs[key]["created_by"] = env.get("created_by", "").strip() or "N/A"
+        grouped_envs[key]["env_type"] = env.get("env_type", "").strip() or "N/A"
+        grouped_envs[key]["state"] = env.get("state", "").strip() or "N/A"
+        if env["account"].strip():
+            grouped_envs[key]["accounts"].add(env["account"].strip())  # Use set to remove duplicates
+
+    # Convert accounts set to a comma-separated string (or "N/A" if empty)
+    for key in grouped_envs:
+        grouped_envs[key]["accounts"] = ", ".join(sorted(grouped_envs[key]["accounts"])) if grouped_envs[key]["accounts"] else "N/A"
+
+    # Define column headers
+    headers = ["NAME", "TEAM", "CREATED BY", "ENV TYPE", "STATE", "ACCOUNTS"]
+
+    # Determine column widths dynamically
+    col_widths = [
+        max(len(key) for key in list(grouped_envs.keys()) + [headers[0]]),
+        max(len(row["team"]) for row in list(grouped_envs.values()) + [{"team": headers[1]}]),
+        max(len(row["created_by"]) for row in list(grouped_envs.values()) + [{"created_by": headers[2]}]),
+        max(len(row["env_type"]) for row in list(grouped_envs.values()) + [{"env_type": headers[3]}]),
+        max(len(row["state"]) for row in list(grouped_envs.values()) + [{"state": headers[4]}]),
+        max(len(row["accounts"]) for row in list(grouped_envs.values()) + [{"accounts": headers[5]}]),
+    ]
+
+    # Print the header row
+    header_format = " | ".join(f"{{:<{w}}}" for w in col_widths)
+    print(header_format.format(*headers))
+    print("-" * (sum(col_widths) + len(col_widths) * 3 - 3))  # Print separator
+
+    # Print each grouped row
+    for name, details in grouped_envs.items():
+        print(header_format.format(name, details["team"], details["created_by"], details["env_type"], details["state"], details["accounts"]))
+
+
+def main():
+    global odin_access_key, odin_secret_access_key, odin_access_token, odin_backend_address, OLD_ODIN
+    # If /opt/homebrew/bin/odin exists, use this as old_odin
+    if os.path.isfile("/opt/homebrew/bin/odin"):
+        OLD_ODIN = "/opt/homebrew/bin/odin"
+
+    if len(sys.argv) == 1:
+        print("No arguments provided")
+        sys.exit(1)
+
+    if len(sys.argv) == 2 and "--version" in sys.argv:
+        execute_old_odin()
+
+    if len(sys.argv) == 2 and "version" in sys.argv:
+        execute_new_odin()
+
+    if "configure" in sys.argv:
+        config_file = os.path.expanduser("~/.odin/config")
+
+        # Delete ~/.odin/config
+        if os.path.isfile(config_file):
+            os.remove(config_file)
```

Comment:
Why are we removing config file? Keep this as it is and create `config.toml` for new odin.